cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -208,6 +214,10 @@ func textFormatPrintMetric(obj *api.Metric) {
 	fmt.Printf("%s: %s | Expire: %s\n", peer.IDB58Encode(obj.Peer), obj.Value, date)
 }
 
+func textFormatPrintRepoGC(obj api.IPFSRepoGC) {
+	fmt.Printf("%s | Error : %s\n", obj.Key.String(), obj.Error)
```

Comment:
Don't print an error tag when there is no error. Look at other places with types containing errors. Keep consistency.